src/Tribe/Main.php

```diff
@@ -3807,17 +3807,25 @@ public function isEvent( $event ) {
 		 * @return bool Is it a venue?
 		 */
 		public function isVenue( $postId = null ) {
+			static $is_venue = [];
+
 			if ( $postId === null || ! is_numeric( $postId ) ) {
 				global $post;
 				if ( isset( $post->ID ) ) {
 					$postId = $post->ID;
 				}
 			}
+
+			// Return if we've already fetched this info.
+			if ( isset( $is_venue[ $postId ] ) ) {
+				return $is_venue[ $postId ];
```

Comment:
:no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"

<sub>hash: 03804ca5d877e2a31318a654544b18ca</sub>

Reply:
Ignoring.